fairlearn/metrics/_roc_auc.py

```diff
@@ -0,0 +1,328 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+import matplotlib.pyplot as plt
+
+from sklearn.metrics import roc_curve, auc, roc_auc_score
+from sklearn.metrics import RocCurveDisplay
+
+from fairlearn.metrics import MetricFrame
+
+class RocAuc:
+    """
+    Provides utilties for generating AUC scores, ROC curves
+    and plotting ROC curves grouped by sensitive feature. Depends on
+    the Fairlearn MetricFrame class to split the input by sensitive feature.
+    Uses the standard Scikit-learn modules `roc_curve`, `auc`, `roc_auc_score`
+    to compute the false positive rate, true positive rate and area under the
+    curve (AUC) respectively. Passing keyword arguemnts to class methods will
+    pass those parameters to the underlying Scikit-learn function to allow for
+    greater control.
+
+    Reference
+    ----------
+    MetricFrame:
+        https://fairlearn.org/v0.6.2/api_reference/fairlearn.metrics.html
```

Comment:
This should be a reference to the class in the docs, not a full URL

Reply:
Yep. Will fix when I run the documentation (I didn't run Sphinx yet.)